### PR TITLE
Add docker-assets and update Dockerfile

### PR DESCRIPTION
**Pull request by lindgrenj6**

### CODE REVIEW

**Review by syncrou — APPROVED** [review]
LGTM 👍

File: Dockerfile [hunk 1]
```diff
@@ -22,8 +27,13 @@ RUN echo "gem: --no-document" > ~/.gemrc && \
...
+COPY docker-assets/run_persister /usr/bin
 
 RUN chgrp -R 0 $WORKDIR && \
     chmod -R g=u $WORKDIR
 
-ENTRYPOINT ["bin/topological_inventory-scheduler"]
+EXPOSE 9394
+
+ENTRYPOINT ["entrypoint"]
+CMD ["run_persister"]
```
Comment (slemrmartin) [inline Dockerfile]: `run_scheduler` ;)

File: Dockerfile [hunk 2]
```diff
@@ -22,8 +27,13 @@ RUN echo "gem: --no-document" > ~/.gemrc && \
     rm -rvf /root/.bundle/cache
 
 COPY . $WORKDIR
+COPY docker-assets/entrypoint /usr/bin
+COPY docker-assets/run_persister /usr/bin
```
Comment (slemrmartin) [inline Dockerfile]: `run_scheduler`

File: Dockerfile (hunk 1 above)
Comment (lindgrenj6) [reply Dockerfile]: you got me!

File: Dockerfile (hunk 2 above)
Comment (lindgrenj6) [reply Dockerfile]: fixed.

**Review by slemrmartin — APPROVED** [review]

**Pull request merged**